mailarchive/mailarchive-subetha/src/java/org/sakaiproject/mailarchive/SakaiMessageHandlerFactory.java

```diff
@@ -684,6 +684,7 @@ else if (p.isMimeType("text/enriched") && p.getFileName() == null) {
                 name = decodedName;
             }
 
+            type=type.replace("[", "").replace("]", ""); //ContentType throws an Exception when parsing these chars. 
```

Comment:
The example content type isn't good even if you remove the `[` and `]`. I think a fuller solution would be to have the `new ContentType` in it's own `try{} catch{}` block and falling back to `application/octet-stream`